samples/recognize.v1p1beta1.js

```diff
@@ -132,6 +132,58 @@ function syncRecognizeModelSelectionGCS(
   // [END speech_transcribe_model_selection_gcs]
 }
 
+function syncRecognizeWithAutoPunctuation(
+  filename,
+  encoding,
+  sampleRateHertz,
+  languageCode
+) {
+  // [START speech_transcribe_file_with_auto_punctuation]
+  // Imports the Google Cloud client library
+  const fs = require('fs');
+  const speech = require('@google-cloud/speech');
+
+  // Creates a client
+  const client = new speech.SpeechClient();
+
+  /**
+   * TODO(developer): Uncomment the following lines before running the sample.
+   */
+  // const filename = 'Local path to audio file, e.g. /path/to/audio.raw';
+  // const encoding = 'Encoding of the audio file, e.g. LINEAR16';
+  // const sampleRateHertz = 16000;
+  // const languageCode = 'BCP-47 language code, e.g. en-US';
+
+  const config = {
+    encoding: encoding,
+    languageCode: languageCode,
+    enableAutomaticPunctuation: true,
+  };
+  const audio = {
+    content: fs.readFileSync(filename).toString('base64'),
```

Comment:
Out of curiosity, does it also work if you do this?

```js
const audio = {
  content: {
    filename: filename // <--- the nodejs libraries sometimes check for the `filename` key and
  }                    // will do the readFileSync/base64 for the user (may or may not work here)
};
```

If so, cool, use that. If not, also cool.

Reply:
Just tried it, it did not work (although I wish it did).    :)